config/config.yaml

```diff
@@ -339,6 +337,12 @@ clouds:
       personal-dev:
         # this is the personal DEV environment
         defaults:
+          #global:
```

Comment:
What are these commented sections for?

Also is a bit confusing to me to have `defaults.*` at the top level and `defaults.*` here

Reply:
each level has their own defaults section, so top level, cloud, env
this format was defined by MSFT